modules/service/src/main/scala/lucuma/odb/graphql/table/GmosDynamicTables.scala

```diff
@@ -15,7 +16,7 @@ trait GmosDynamicTables[F[_]] extends BaseMapping[F] {
     grating: skunk.Codec[G],
     filter:  skunk.Codec[L],
     fpu:     skunk.Codec[U]
-  ) extends TableDef(name) {
+  )(implicit gratingName: TypeName[G], filterName: TypeName[L], fpuName: TypeName[U]) extends TableDef(name) {
```

Comment:
It's just to please the validation checker, which relies on capturing the compile-time types for `ColumnRef`s and other things. In this case it was capturing type variables instead of the actual type names, which was causing bogus validation errors.